.storybook/manager-head.html

```diff
@@ -0,0 +1,56 @@
+<style>
+    .sidebar-header {
+        position: fixed !important;
+        width: 210px;
+        background-color: #fff;
+        border-bottom: 2px solid #d7dbde;
+        z-index: 10;
+        padding: 21px 16px 15px 20px !important;
+        top: 0;
+        left: 0;
+    }
+    
+    .sidebar-header > div {
+        position: relative;
+    }
+
+    .sidebar-header > div:first-child {
+        position: relative;
+        margin-right: 0;
+    }
+
+    .sidebar-header > div:last-child {
+        display: none;
```

Comment:
Per our discussion on slack, I think we need to keep this

Reply:
😢

Reply:
ok done.

Reply:
I know I know it looks so much better with it hidden 😿